plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableQuery.php

```diff
@@ -1356,13 +1365,35 @@ private function run_query(): void {
 		}
 
 		if ( $this->limits ) {
-			$this->found_orders  = absint( $wpdb->get_var( $this->count_sql ) ); // phpcs:ignore WordPress.DB.PreparedSQL.NotPrepared
+			$this->found_orders  = $this->get_count();
 			$this->max_num_pages = (int) ceil( $this->found_orders / $this->args['limit'] );
 		} else {
 			$this->found_orders = count( $this->orders );
 		}
 	}
 
+	/**
+	 * Get the order count by cache or SQL.
+	 */
+	private function get_count() {
+		global $wpdb;
+
+		$where_string  = substr( $this->count_sql, strpos( $this->count_sql, 'WHERE 1=1 AND' ) + 14, -1 );
```

Comment:
This needs to be cleaned up and account for other clauses that might follow the `WHERE` clause.